README.md

```diff
@@ -32,69 +36,145 @@ Create `exams.sql` file with `INSERT INTO` query that adds entries to `exams` ta
 Write a query that outputs `year`, `modifier`, `student_cnt` columns that specify the number of students (`student_cnt`) in a class for all classes.
 
 ```sql
-PASTE YOUR CODE HERE
+SELECT c.year, c.modifier, count(*) as student_cnt  FROM class as c
+JOIN groups
+ON id=class_id
+GROUP by year, modifier
+
 ```
 
 ### 2. Find the number of students for every year
 
 Write a query that outputs `year`, `student_cnt` columns that specify the number of students (`student_cnt`) in a class for each year (`year`).
 
 ```sql
-PASTE YOUR CODE HERE
+SELECT c.year, count(*) as student_cnt  FROM class as c
+JOIN groups
+ON id=class_id
+GROUP by year
 ```
 
 ### 3. Find unique subjects that students learn for a given year
 
 Write a query that outputs `name`, `surname`, `unique_lesson_cnt`, where `unique_lesson_cnt` is the number of **unique** lessons that a student identified by `name`, `surname` has.
 
 ```sql
-PASTE YOUR CODE HERE
+SELECT c.year,s.name, s.surname, count(*) AS unique_lesson_cnt FROM lessons as l
```

Comment:
Grouping by name and surname is not correct. What if there are namesakes (i.e. students with the same name and surnames, but different IDs)?